packages/client/hmi-client/src/model-representation/petrinet/petrinet-renderer.ts

```diff
@@ -1,8 +1,24 @@
+import _ from 'lodash';
 import * as d3 from 'd3';
 import { BasicRenderer, INode, IEdge } from '@graph-scaffolder/index';
 import { D3SelectionINode, D3SelectionIEdge } from '@/services/graph';
 import { pointOnPath } from '@/utils/svg';
-import { NodeData, EdgeData, NodeType } from './petrinet-service';
+import { Model } from '@/types/Types';
+import * as petrinetService from '@/model-representation/petrinet/petrinet-service';
+
+export interface NodeData {
+	type: string;
+	uid?: string | number;
```

Comment:
Is this a hold-over to handle legacy data or are uids actually strings and numbers?

Reply:
That is a copy and pasta mistake from the old petri-net renderer. Will remove.